utils/dc-chain/scripts/gcc-pass1.mk

```diff
@@ -5,9 +5,9 @@
 # Initially adapted from Stalin's build script version 0.3.
 #
 
-$(build_gcc_pass1) $(build_gcc_pass2): build = build-gcc-$(target)-$(gcc_ver)
+$(build_gcc_pass1): build = build-gcc-$(target)-$(gcc_ver)-pass1
 $(build_gcc_pass1) $(build_gcc_pass2): src_dir = gcc-$(gcc_ver)
-$(build_gcc_pass1): log = $(logdir)/$(build)-pass1.log
+$(build_gcc_pass1) $(build_gcc_pass2): log = $(logdir)/$(build).log
```

Comment:
If I understand well, you want to have both pass1 and pass2 in the same log file?
Interesting approach but I think it could be interesting to have a clear separation in the log file to distinguish pass1 and pass2. What do you think?

Reply:
This actually creates 2 logs still, because the build variable name is being changed to include the pass number. If we keep the log variable as-is then the logs are named `build-gcc-sh-elf-9.3.0-pass1-pass1.log` and `build-gcc-sh-elf-9.3.0-pass2-pass2.log`.

Reply:
Thanks for the explanation, maybe there is something easy to do for avoiding pass keyword repeating?

Reply:
Sorry I believe I worded my reply a bit oddly. With the proposed changes the following logs files for GCC would be output:
```
build-gcc-sh-elf-9.3.0-pass1.log
build-gcc-sh-elf-9.3.0-pass2.log
```
If line 10 above was not modified then it would produce the double pass keyword output files mentioned above. I simply was trying to say that my intention with the line 10 change was to fix the double keyword in the file name and not to combine the log files. I think having them separate makes the most sense as well :slightly_smiling_face: 

Please do correct me if that was not the pass keyword repetition you're talking about